### ISSUE TITLE
Customizer: Reply to comment warning

### ISSUE THREAD

**Issue by eri-trabiccolo** [presscustomizr/customizr#545]
When in the Customizer and clicking on a comment Reply link you'll see this warning (wp_debug enabled):
Notice: Trying to get property of non-object in /xxx/wp-includes/comment-template.php on line 646

This doesn't happen with twentysixteen ..

**Comment by eri-trabiccolo** [comment]
Mmm
basically the thing is that in that comment-template function the comment object is empty.
I've been able to reproduce the same issue with twentysixteen just placing the `comment_form()` call before the eventual printing of the list of comments.
With 2016 happens in front too (not only in the customizer)
It doesn't happen in "our" front 'cause our reply is handled via js. In the customizer there's an old known - or at least known by me :D - bug when you click on a link with an anchor which makes reload the page anyways, so we get this issue.
Have to look what's the code when printing the comment list which basically initializes the global comment variable.

So basically it's a matter of being "outside the comments loop", if we can say so...

(Hope I've been understandable :D)

**Comment by eri-trabiccolo** [comment]
Forgot to keep you updated on this.
Basically it's due to a wp bug.
Of course I have a solution for "us" till they fix it but.. I don't know if it's even worthwhile since it happens in edge cases:
- trying to reply to a comment with js disabled
or
- trying reply to a comment in the cusomizer preview, which anyway is a nonsense as it will still not be submitted (customizer preview thing, not our fault :) ).

p.s.
https://core.trac.wordpress.org/ticket/35624#comment:4
